app/src/app/[lng]/data/review/page.tsx

```diff
@@ -54,13 +70,16 @@ export default function ReviewPage({
   };
 
   const [addUserFile] = api.useAddUserFileMutation();
+  const [sendEmailNotification, { isLoading: isSendingLoading }] =
+    api.useSendEmailNotificationMutation();
```

Comment:
This should not happen in the frontend.
The user should not have an endpoint to send random emails to our team.

Rather, this should be a function that's called when a file finished uploading on the backend. Also not on the review page, but immediately when it's done uploading.